### PR TITLE
 Fix for Client Id Config Bug

### PR DESCRIPTION
**Pull request by voduyemi**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/classes/PS_IntegrationServiceConfig_TEST.cls [hunk 1]
```diff
@@ -96,18 +98,30 @@ public with sharing class PS_IntegrationServiceConfig_TEST {
...
+                .withProductId(false)
+                .withActiveFlagAsFalse(true);
+        PS_IntegrationServiceConfig.paymentServicesConfigurationSelector = stubFor(configurationSelectorMock);
         PS_IntegrationServiceConfig ps = new PS_IntegrationServiceConfig();
         System.assertEquals(false, ps.isIntegrationEnabled(),
             'The org should have Integration Disabled when isActive record is false');
     }
 
+    @IsTest
+    private static void shouldHaveIntegrationDisabledWhenIsActiveServiceRecordIsTrue() {
```
Comment (kenlewis-sfdo) [inline force-app/main/default/classes/PS_IntegrationServiceConfig_TEST.cls:113]: Should this one be "Enabled" instead of disabled (in the method name)?

Comment (voduyemi) [reply force-app/main/default/classes/PS_IntegrationServiceConfig_TEST.cls]: Pushed up an update

**Review by psadie-sfdo — APPROVED** [review]
Looks good! Thanks, Victor!

**Review by kenlewis-sfdo — APPROVED** [review]
Thanks for the update and the thorough new tests, Victor!

**Pull request merged**